cmd/knoxite/store.go

```diff
@@ -54,11 +52,32 @@ var (
 			if len(args) < 2 {
 				return fmt.Errorf("store needs to know which files and/or directories to work on")
 			}
+			storeOpts = configureStoreOpts(cmd, storeOpts)
+
 			return executeStore(args[0], args[1:], storeOpts)
 		},
 	}
 )
 
+// configureStoreOpts will compare the the setting from the configration file and
```

Comment:
typo: dupe "the".

Reply:
Fixed the dupe and also the typo in `configuration`